public/app/features/datasources/components/DataSourcesList.tsx

```diff
@@ -102,5 +119,8 @@ const getStyles = () => {
     logo: css({
       objectFit: 'contain',
     }),
+    button: css({
+      marginLeft: '16px',
```

Comment:
I think it would be neater to use `theme.spacing(2)` here over hardcoded values.

Reply:
Done